docs/user/advanced.rst

```diff
@@ -940,7 +937,7 @@ If the remote server is very slow, you can tell Requests to wait forever for
 a response, by passing None as a timeout value and then retrieving a cup of
 coffee.
 
-.. code-block:: python
+::
```

Comment:
One more note on this. It looks it does affect the GitHub highlighting but not python-requests.org. 

The two previous code snippets above this one do not currently display with any syntax highlighting, but this one does. I don't have a strong opinion on which direction to go with this, but I do think it looks a little odd if highlighting isn't uniform across grouped examples.